packages/behandling-omsorgspenger/src/types/fetchedDataTsType.ts

```diff
@@ -18,6 +18,7 @@ interface FetchedData {
   beregningsgrunnlag: Beregningsgrunnlag;
   simuleringResultat: SimuleringResultat;
   forbrukteDager: ÅrskvantumForbrukteDager;
+  tilgjengeligeVedtaksbrev?: string[];
```

Comment:
Hadde det vært mulig å få backend til å sende en tom array i stedet for ingenting, så kunne vi ha fjernet ?

Reply:
Det gjør den allerede, så jeg kan fjerne `?` herfra.

Reply:
Gjør den så er vel denne alltids tilgjengeligeVedtaksbrev en array og gjør denne sjekken unødvendig !Array.isArray(tilgjengeligeVedtaksbrev) da den alltid vil gi "!true"

Reply:
Nei. Sjekken ligger i VedtakForm, som også brukes av andre komponenter. Dette skal utvides, men per nå er det kun omsorgspenger som sender tilgjengeligeVedtaksbrev til VedtakForm.

Reply:
Skjønner